http-gateways/DNSLINK_GATEWAY.md

```diff
@@ -0,0 +1,94 @@
+# DNSLink Gateway Specification
+
+![](https://img.shields.io/badge/status-wip-orange.svg?style=flat-square)
+
+**Authors**:
+
+- Marcin Rataj ([@lidel](https://github.com/lidel))
+
+----
+
+**Abstract**
+
+DNSLink Gateway is an extension of
+[PATH_GATEWAY.md](./PATH_GATEWAY.md)
+that enables hosting a specific content path under a specific DNS name.
+
+This document describes the delta between [PATH_GATEWAY.md](./PATH_GATEWAY.md) and this gateway type.
+
+In short:
+
+- HTTP request includes a valid [DNSLink](https://dnslink.dev/) name in `Host` header
+- gateway resolves DNSLink to an immutable content root identified by a CID
+- HTTP response includes the data for the CID
+- No third-party CIDs can be loaded
```

Comment:
At least up to this point reading this document, I don't know what is meant by "third-party CID".

Reply:
simplified in ce193e6